src/network/webtask.h

```diff
@@ -182,6 +182,11 @@ class WebTask : public NetworkTask {
             const QString& errorString,
             const WebResponseWithContent& responseWithContent);
 
+  protected:
+    virtual void doNetworkError(
```

Comment:
The `do` prefix indicates the *Template Method* pattern with a pure virtual function that is only invoked by the base class and inaccessible for derived classes. Derived classes are supposed to implement those functions, but are not allowed to invoke them at arbitrary times.

In contrast this seems to be an ordinary, overridable virtual function. It is also unclear even clear why and when you should override it?

Reply:
What is you suggestion to improve this situation.

Reply:
O have renamed onNetworkError() to doNetworkError()